catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/validation/impl/PatternValidatorTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.validation.impl;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.validation.impl.validator.PatternValidator;
+import ddf.catalog.validation.report.AttributeValidationReport;
+
+public class PatternValidatorTest {
+    private static final String REGEX = "(?i)^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,}$";
```

Comment:
To keep the test simple, we could assume that the `PatternValidator` delegates internally to Java's `Matcher` and use a simpler regex. We're not trying to test that `Matcher` works in this case, just making sure that our code behaves as expected when the `Matcher` returns `true` and `false`.